docs/connector-development/connector-builder-ui/tutorial.mdx

```diff
@@ -62,6 +60,8 @@ Your connector builder project is now set up. The next steps describe how to con
 
 ### Setting up global configuration
 
+<div style={{ position: "relative", paddingBottom: "59.66850828729282%", height: 0 }}><iframe src="https://www.loom.com/embed/ee170ac0431f46e297fbb2aa571fc9de" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style={{position: "absolute", top: 0, left: 0, width: "100%", height: "100%"}}></iframe></div>
```

Comment:
nit: might be worth re-recording this one without the typos since it's only 16 seconds

Reply:
Recreated this one